### PR TITLE
hypershift, kubevirt: Add optional presubmit conformance job

### PR DESCRIPTION
**Pull request by qinqon**

### CODE REVIEW

File: ci-operator/jobs/openshift/hypershift/openshift-hypershift-main-presubmits.yaml [hunk 1]
```diff
@@ -598,6 +598,85 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )e2e-kubevirt-aws-ovn,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
```
Comment (orenc1) [inline ci-operator/jobs/openshift/hypershift/openshift-hypershift-main-presubmits.yaml:602]:
```suggestion
    always_run: false
```

Comment (qinqon) [reply ci-operator/jobs/openshift/hypershift/openshift-hypershift-main-presubmits.yaml:602]: I have add `always_run: false` to the config but the job is nog generated with false.

**Review by davidvossel — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**